code/modules/clothing/rogueclothes/mask.dm

```diff
@@ -118,6 +118,38 @@
 	max_integrity = 75
 	smeltresult = /obj/item/ingot/gold
 
+/obj/item/clothing/mask/rogue/facemask/leper
+	name = "steel leper mask"
+	desc = "A gem-dust enchanted steel mask, Those are used to suppress the spasms of leprosy... But can hardly see in it.."
+	icon = 'modular_stonehedge/icons/roguetown/clothing/masks.dmi'
+	mob_overlay_icon = 'modular_stonehedge/icons/roguetown/clothing/onmob/masks.dmi'
+	icon_state = "steelmaskleper"
+	block2add = FOV_BEHIND
+	max_integrity = 200 //same as steel mask
+
+//Cant see a fucking thing.
+/obj/item/clothing/mask/rogue/facemask/leper/equipped(mob/living/carbon/human/user)
+	. = ..()
+	if(user.wear_mask)
+		user.apply_status_effect(/datum/status_effect/debuff/lepermask)
+
+/obj/item/clothing/mask/rogue/facemask/leper/attackby(obj/item/I, mob/living/carbon/human/user, params) //for removing into hand check
+	. = ..()
+	if(!user.wear_mask)
+		user.remove_status_effect(/datum/status_effect/debuff/lepermask)
+
+/obj/item/clothing/mask/rogue/facemask/leper/dropped(mob/living/carbon/human/user)
+	..()
+	if(!user.wear_mask)
+		user.remove_status_effect(/datum/status_effect/debuff/lepermask)
+
+/obj/item/clothing/mask/rogue/facemask/leper/gold
+	name = "gold leper mask"
+	desc = "A gem-dust enchanted gold mask, Those are used to suppress the spasms of leprosy... But can hardly see in it.."
```

Comment:
Fix the last ssentence here, But you can hardly see in it, or something else, just like this isn't good

Reply:
things are usually 'in your head' sorta so